script/import_scripts/yahoogroup.rb

```diff
@@ -0,0 +1,150 @@
+require File.expand_path(File.dirname(__FILE__) + "/base.rb")
+require 'mongo'
+
+# Import YahooGroups data as exported into MongoDB by:
+#   https://github.com/jonbartlett/yahoo-groups-export
+
+class ImportScripts::YahooGroup < ImportScripts::Base
+
+  def initialize
+    super
+
+    client = Mongo::Client.new([ '192.168.10.1:27017'], :database => 'syncro')
```

Comment:
These should be constants and/or environment variables.
I suggest doing something like this: https://github.com/discourse/discourse/blob/master/script/import_scripts/mbox.rb#L22-L25